### PR TITLE
increase timeout

### PR DESCRIPTION
**Pull request by ubozov**

### CODE REVIEW

File: util/misc.go [hunk 1]
```diff
@@ -24,8 +24,8 @@ import (
 )
 
 const (
-	//Timeout - 90 sec
-	Timeout time.Duration = 90 * time.Second
+	//Timeout - 3 min
+	Timeout time.Duration = 3 * time.Minute
```
Comment (dzeckelev) [inline util/misc.go]: Can transfer it in the configuration file?

Comment (ubozov) [reply util/misc.go]: updated

File: dapp/dapp.go [hunk 2]
```diff
@@ -33,6 +33,7 @@ type Dapp struct {
 	Version    string
 	Tor        *tor.Tor
 	UserID     string
+	Timeout    time.Duration
```
Comment (dzeckelev) [inline dapp/dapp.go]: 1 sec is time.Duration(1000000000). 
Can we use seconds or milliseconds?

Comment (ubozov) [reply dapp/dapp.go]: updated, using seconds

**Pull request merged**